### PR TITLE
LPS-174874 Use SAXReaderUtil to format xml string instead of Dom4jUtil

### PR DESCRIPTION
**Pull request by julschong**

### CODE REVIEW

**Comment by zhangyan0701** [comment]
ci:test:usm

File: modules/apps/user-associated-data/user-associated-data-api/src/main/java/com/liferay/user/associated/data/exporter/DynamicQueryUADExporter.java [hunk 1]
```diff
@@ -118,7 +120,13 @@ protected File createFolder(long userId) {
 
 	protected String formatXML(String xml) {
 		try {
-			return Dom4jUtil.toString(xml);
+			xml = StringUtil.replace(xml, "]]><", "[$SPECIAL_CHARACTER$]");
+			xml = StringUtil.replace(xml, "]]>", "]]]]><![CDATA[>");
+			xml = StringUtil.replace(xml, "[$SPECIAL_CHARACTER$]", "]]><");
+
```
Comment (julschong) [inline modules/apps/user-associated-data/user-associated-data-api/src/main/java/com/liferay/user/associated/data/exporter/DynamicQueryUADExporter.java:126]: add in logic from Dom4jUtil to deal with ]]> characters

**Comment by julschong** [comment]
ci:test:sf

**Comment by julschong** [comment]
ci:test:relevant

**Comment by julschong** [comment]
ci:test:usm

**Pull request closed without merging**